lib/is-element.test.js

```diff
@@ -3,11 +3,9 @@
 var assert = require("@sinonjs/referee").assert;
 
 // in order for `is-element.js` to correctly detect the DOM being set up
-// in the previous line, we need to reload it, as other tests will have
+// with `jsdom-global`, we need to reload it, as other tests will have
```

Comment:
Fixed the same thing [here](https://github.com/sinonjs/samsam/blob/12bf8027d93e2cbaac9fc141db929197e107b781/lib/is-element.test.js). This would now conflict 😕 How can we proceed?

Reply:
It's not a big conflict. I like your version better as that doesn't create new variables